components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -298,67 +302,89 @@ export const Account = ({
         </AccountButtonsRow>
       </WalletInfoRow>
 
-      <SubviewSectionTitle>{getLocale('braveWalletAccountsAssets')}</SubviewSectionTitle>
-
-      <SubDivider />
       <Spacer />
-
-      {fungibleTokens.map((item) =>
-        <PortfolioAssetItem
-          key={`${item.contractAddress}-${item.symbol}-${item.chainId}`}
-          assetBalance={getBalance(selectedAccount, item)}
-          token={item}
-          isAccountDetails={true}
-          showSellModal={() => onClickShowSellModal(item)}
-          isSellSupported={checkIsAssetSellSupported(item)}
-        />
-      )}
-
-      {!assetAutoDiscoveryCompleted &&
-        <PortfolioAssetItemLoadingSkeleton />
-      }
-
       <Spacer />
```

Comment:
Would our re-useable VerticalSpacer component work here so that we can deprecate this style if unused elsewhere?

Reply:
We can probably handle this kind of stuff when we start working on `V2` implementation IMO. Goal of this PR was to get the existing screens working in the `Box Container`, what do you think?